### DISCUSSION TITLE
Specifying the logging level using the Console.log() API

### DISCUSSION THREAD

**Discussion by byt3bl33d3r** [Textualize/rich#510] [Q&A] [answered]
Hey there,

Amazing library, kudos. 

I was wondering if there is a way of specifying a logging level when calling `console.log()` ? From the API docs it doesn't seem there is. This is basically the only thing that's stopping me from using the Console object instead of the `RichHandler` as of right now as it's sort of essential when dealing with large applications.

Are there any plans on adding a way of specifying a log level to the `console.log()` method? or is it not meant as replacement for the logging module ?

Thanks & Cheers

**Comment by willmcgugan** [comment, marked as answer]
Glad you like Rich.

There's no level parameter in console.log.

`console.log()` was not meant as a replacement for logging, even if they do generate similar output. The log method was more for quick debug output similar to Javascript's console.log.

**Reply by byt3bl33d3r** [reply]
Thank you!

Cheers
